news/2018-03-13-project-loved-week-of-march-11th.md

```diff
@@ -0,0 +1,244 @@
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/taiko/shippuujinrai.jpg)](https://osu.ppy.sh/community/forums/topics/715048)
+
+A superstar compilation of Japanese mappers difficulties on a single song, lead by [yassu-](https://osu.ppy.sh/users/7095592), [Shippuujinrai](https://osu.ppy.sh/beatmapsets/465330) is a great map for practicing your technical side of play, as well as just having a cavalcade of quality difficulties to enjoy. Also featured in TWC, the difficulties here are usually rife with different styles of 1/6 and interpretations of the music, there is certainly something for most levels of play, even some on the easier side with Osamix's Oni!
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/taiko/enigma.jpg)](https://osu.ppy.sh/community/forums/topics/715049)
+
+Finally! The arrival of the infamous [KinomiCandy](https://osu.ppy.sh/users/375143), with one of his popular YUC'e Ma-...Oh it's not YUC'e? Actually, this is one of his more unique maps, with a piano heavy track that's unlike the usual poppy, happy tracks that he has become popular for. We simply chose this track because of it's uniqueness to his catalogue, but certainly some of his YUC'e maps will hit Loved in the future. Kinomi's style of mapping lends itself well to the ever-growing catalogue of Loved!
+
+## <a name="osucatch" id="osucatch"></a>osu!catch
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/gJ85pqQ2f6Q?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+---
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/catch/amphisbaena.jpg)](https://osu.ppy.sh/community/forums/topics/715055)
+
+*written by [Deif](https://osu.ppy.sh/users/318565)*
+
+A quite difficult challenge presented by one of the most memorable NoMod players. [Amphisbaena](https://osu.ppy.sh/beatmapsets/511738) alternates calm sections with complex hyperdash combinations that'll hopefully twist the mind of the most experienced players.
+
+[![](/wiki/shared/news/2018-03-13-project-loved-week-of-march-11th/catch/ice-angel.jpg)](https://osu.ppy.sh/community/forums/topics/715056)
+
+*written by [Sartan](https://osu.ppy.sh/users/4100941)*
+
+One of the most severly underrated and cleanest maps made by [Arthas](https://osu.ppy.sh/users/3414261) (now known as Razor Sharp) makes a comeback on the loved selection after having been left untouched for almost a year. [This map](https://osu.ppy.sh/beatmapsets/549286) is filled with very straightforward patterning which really compliments the beautiful song that Ice Angel is. It also serves to be quite a challenge to fc, especially when attempted with mods such as hardrock or hidden, so the replayability on this one is great as well. All in all: if you want to be Razor Sharp's little (ice) angel here, go vote!
```

Comment:
`challenge to fc, especially` -> `challenge to FC, especially`
FC is an initialism, so it should be in caps